pkg/webreg/webreg.go

```diff
@@ -1880,37 +1879,310 @@ as the component.
+There is a separate configuration file per branch, and the configuration files
+follow the <code>$org-$repo-$branch.yaml</code> pattern:
+</p>
+
+{{ plaintextSyntax (index . "ciopConfigDirExample") }}
+
+<p>
+For the repositories involved in the <a href="https://docs.google.com/document/d/1USkRjWPVxsRZNLG5BRJnm5Q1LSk-NtBgrxl2spFRRU8/edit#heading=h.3myk8y4544sk">Centralized Release Branching and Config Management</a>,
+(most of the OpenShift components) the configuration for release branches for the
+<em>future</em> releases are managed by automation and should not be changed or
+added by humans.
+</p>
+
+<h5 id="feature-branches"><a href="#feature-branches">Feature Branches</a></h5>
+
+<p>
+Any branch whose name has a prefix matching to any branch that has a <code>ci-operator</code>
+configuration file is considered a <em>"feature branch"</em>. Pull requests to
+feature branches trigger the same CI presubmit jobs (but not postsubmits) like
+configured for the base branch, without any additional configuration. This also
+means that such <em>"feature branches"</em> cannot have a separate, different
+<code>ci-operator</code> configuration. For example, if a repo has a <code>org-repo-release-2.0.yaml</code>
+config (specifying CI config for the <code>release-2.0</code> branch of that
+repository), the same CI presubmits will trigger on pull requests to a <code>release-2.0.1</code>
+branch, and the repo cannot have a <code>org-repo-release-2.0.1.yaml</code>
+configuration file.
+</p>
+
+<h5 id="variants"><a href="#variants">Variants</a></h5>
+
+<p>
+It is possible to have multiple <code>ci-operator</code> configuration files for
+a single branch. This is useful when a component needs to be built and tested in
+multiple different ways from a single branch. In that case, the additional
+configuration files must follow the <code>org-repo-branch__VARIANT.yaml</code>
+(note the double underscore separating the branch from the variant).
+</p>
+
+<h4 id="component-jobs"><a href="#component-jobs">Prowjob Configuration</a></h4>
+
+<p>
+Most jobs are generated from the<code>ci-operator</code> configuration, so the
+need to interact with actual Prowjob configuration should be quite rare.
+Modifying the Prowjob configuration is discouraged unless really necessary, and
+can result in increased fragility and maintenance costs.
+</p>
+
+<h5 id="manual-job-changes"><a href="#manual-job-changes">Tolerated Changes to Generated Jobs</a></h5>
+
+<p>
+Generated jobs are enforced to stay in the generated form, so when you attempt
+to change them, a check will fail on the pull requests, requiring the jobs to be
+regenerated and changed back. However, the generator tolerates these
+modifications to allow some commonly needed customizations:
+</p>
+
+<ul>
+    <li>
+        <code>.always_run</code>: can be set to
+        <code>false</code> to disable automated triggers of a job on pull requests (presubmits)
```

Comment:
Could this be a table:

| Field         | Description                                                                | Presubmit          | Postsubmit | Periodic |
|---------------|----------------------------------------------------------------------------|--------------------|------------|----------|
| `.always_run` | <code>false</code> to disable automated triggers of a job on pull requests | :heavy_check_mark: |            |          |
|               |                                                                            |                    |            |          |
|               |                                                                            |                    |            |          |